src/recordlinker/linkage/algorithms.py

```diff
@@ -25,30 +25,30 @@
 DIBBS_BASIC = [
```

Comment:
What is the purpose of having the algorithms defined in multiple places (both `src/recordlinker/linkage/algorithms.py` and `assets/linkage/`)?

Reply:
Honestly, not much.  There are some test cases that are using the assets directory for validation, but that could easily be done using `json.dumps` + the contents of algorithms.py.  Long term, many of these test cases will be deleted because the work Cason is doing with moving the algorithms to the DB will make them irrelevant.